src/olympia/reviewers/models.py

```diff
@@ -1132,6 +1132,9 @@ class ReviewActionReason(ModelBase):
     )
     name = models.CharField(max_length=255)
 
+    def labelled_name(self):
+        return self.name + ' (** inactive **)' if not self.is_active else self.name
```

Comment:
I'd go with prepending `(inactive) ` rather than appending, thinking about it - `name` can be up to 255 long so it could be easy to overlook.